### ISSUE TITLE
US3 Records of the game

### ISSUE THREAD

**Issue by WalterRestifo** [WalterRestifo/capstone-project#3]

**Comment by AntjeWalter** [comment]
I think this user story is pretty big. Would it be possible to split this up and make more simple user stories from it? 
Furthermore, you forgot to add tasks.

**Comment by WalterRestifo** [comment]
Yes it is a big story, but it is not a problem if I need two days to complete it instead of one. It is difficult to have an app that is working at the end of every user story... I cannot make the main page in a story and and the new game in another one, without having buttons that don´t do anything. I would let this story size like this.
I didn´t forget to write tasks, I just don´t exactly know how to let it work. I probably need to save the data of the points and games somewhere and I need to code the pages, but I will think about it in the development.
